src/jit/liveness.cpp

```diff
@@ -1103,6 +1103,13 @@ void Compiler::fgExtendDbgLifetimes()
             {
                 continue;
             }
+#if !defined(_TARGET_64BIT_) && !defined(LEGACY_BACKEND)
+            // On x86, this applies to long-typed variables as well once we've hit LIR. 
```

Comment:
Unfortunately it looks like this comment isn't quite right... I'm investigating.